src/host-callback.js

```diff
@@ -161,7 +161,7 @@ class HostCallback {
    * Returns true iff this task was scheduled with MessageChannel.
    * @return {boolean}
    */
-  isMessageChannelCallback() {
+  isMessageChannelCallback_() {
```

Comment:
This is really just unused (except for tests), but happy to revert since the intention appears for it to be usable in scheduler, similar to `isIdleCallback`

Reply:
Yeah this was only needed for tests, and I vaguely recall being hesitant about adding the bytes. I believe this was added because some places this was used didn't have MessageChannel, and this was helpful for a regression test. But minimizing the impact of this LGTM.